src/util/util.ts

```diff
@@ -83,6 +83,29 @@ function writeFile(path: string, file: string, encoding?: string): any {
   });
 }
 
+const evalReg = new RegExp(/\beval\(([^),]*)\)/g);
+
+// hasEval determine whether matcher contains function eval
+function hasEval(s: string): boolean {
+  return evalReg.test(s);
+}
+
+// replaceEval replace function eval with the value of its parameters
+function replaceEval(s: string, rule: string): string {
+  return s.replace(evalReg, '(' + rule + ')');
+}
+
+// getEvalValue returns the parameters of function eval
+function getEvalValue(s: string): string[] {
+  const subMatch: string[] = s.match(evalReg) as string[];
```

Comment:
Please avoid using `as`.  Maybe `s.match(evalReg)` returns `null` or `undefined`.

Reply:
Okay

Reply:
But in order to iterate through subMatch it will have to type cast it to string[].

Reply:
You can check the type of subMatch is `null` or `undefined`, something like:
```
if (!subMatch) { return []; }
```